polling_stations/apps/addressbase/management/commands/import_cleaned_addresses.py

```diff
@@ -0,0 +1,28 @@
+import os
+import glob
+
+from django.db import connection
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'cleaned_ab_path',
+            help='The path to the cleaned AddressBase file'
```

Comment:
Minor point, but can this be clarified to "The path to the folder containing the cleaned AddressBase CSVs" or similar